spec/features/team/show_team_spec.rb

```diff
@@ -61,4 +61,19 @@
       expect(page).to have_link(I18n.t("helpers.links.email"), href: mailto_link_for_user)
     end
   end
+
+  describe 'display team members only for logged in users' do
+
+    it 'should not display team member table for users who are not logged in ' do
+      sign_out @user
```

Comment:
@tomkellog I would assume `@user` is not signed in, hence there is no need to sign out the `@user`. Can this line be deleted?

Reply:
Or you add the following block and adjust both test cases:
```
before(:each) do
    sign_in @user
end
```